### DISCUSSION TITLE
runit support (void linux init system)

### DISCUSSION THREAD

**Discussion by zmeyka3310** [bol-van/zapret#588] [Ideas]
technically it should be as easy as making a .sh file to start the service from anywhere (as a normal bash script starting it from any place), and then symlinking it into the proper place. https://wiki.archlinux.org/title/Runit#User_Services thats covered properly here. for configuring the service, passing parameters, doing a proper service shutdown, checking if the service is still up and so on, please refer to https://docs.voidlinux.org/config/services/index.html. ill try to do something tomorrow, but meanwhile im posting this here (а еще я по русски разговариваю, хз если кому так удобнее)

**Comment by bol-van** [comment]
There's runit->sysv adapter in /opt/zapret/init.d/runit

**Reply by zmeyka3310** [reply]
thanks
